docs/wcs/examples/from_file.py

```diff
@@ -40,6 +41,22 @@ def load_wcs_from_file(filename):
     # some floating-point error.
     assert np.max(np.abs(pixcrd - pixcrd2)) < 1e-6
 
+    # As illustrated below, origin defines whether the origin pixel position
+    # is 0- or 1-indexed.
```

Comment:
I feel like `origin pixel position` is not clear. Is it clearer to say something like

```
The example below illustrates the use of `origin` to convert between 0- and 1- based coordinates
when executing the forward and backward WCS transform.
```

Reply:
Done.